RocketChat/Rocket.Chat
# [FIX]: Tooltip renderer function

## Proposed changes (including videos or screenshots)


As described in the issue, two different tooltips are appearing on hovering the timestamp, or any tooltip element.
This default tooltip remains for a second in chrome, while it remains for a while in safari.
And the event listener was executing the tooltip function on every cursor movement, which is a bad practice, so, have added a debounce function to call the tooltip, which has reduced the function calls by a lot.



## Issue(s)

Closes #20902 

## Steps to test or reproduce

Hover on the timestamp, two different tooltips will appear.
If we hover on the edge of the element, the tooltip will never be seen, due to a continuous function call which is blocking any other execution.

## Further comments
